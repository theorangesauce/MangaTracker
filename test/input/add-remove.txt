--ADD/REMOVE TEST----
--ADD TEST-----------
A
New Series
1-5, 7
Author 3
New Publisher
Series the New
N
--LIST---------------
L
L
--REMOVE TEST--------
R
new series
Y
Y
L
L
X
--END ADD/REMOVE-----
