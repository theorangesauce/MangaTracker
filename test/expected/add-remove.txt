Add Results:
  List All:
    Last Series
    New Series
    Series 1
    Series 2
    Series 3
    Series 4

Remove Results:
  List All:
    Last Series
    Series 1
    Series 2
    Series 3
    Series 4
